components/proxy/src/main/java/com/hotels/styx/routing/handlers/HostProxy.java

```diff
@@ -287,24 +286,24 @@ private static HostAndPort addDefaultPort(HostAndPort hostAndPort, TlsSettings t
         @NotNull
         public static HostProxy createHostProxyHandler(
                 NettyExecutor executor,
-                MetricRegistry metricRegistry,
+                MeterRegistry meterRegistry,
                 HostAndPort hostAndPort,
                 ConnectionPoolSettings poolSettings,
                 TlsSettings tlsSettings,
                 int responseTimeoutMillis,
                 int maxHeaderSize,
-                String metricPrefix,
-                String objectName) {
+                String appId,
+                String originId) {
```

Comment:
Have a look at the proposal in PR#691 to discuss this further. It's hard to remove completely because Styx has the concept of Origin spread throughout the codebase. But that PR is an attempt to begin removing that.